src/scores/categorical/contingency_impl.py

```diff
@@ -378,54 +381,124 @@ def threat_score(self) -> xr.DataArray:
         ts = cd["tp_count"] / (cd["tp_count"] + cd["fp_count"] + cd["fn_count"])
         return ts
 
-    def critical_success_index(self):
+    def critical_success_index(self) -> xr.DataArray:
         """
-        Often known as CSI.
-
         Identical to threat_score
-        Range: 0 to 1, 0 indicates no skill. Perfect score: 1.
 
-        https://www.cawcr.gov.au/projects/verification/#CSI
+        .. math::
+            \\text{threat score} = \\frac{\\text{true positives}}{\\text{true positives} + \\text{false positives} + \\text{true negatives}}
+
+        Returns:
+            An xarray object containing the critical success index
+
+        Notes:
+            - Range: 0 to 1, 0 indicates no skill. Perfect score: 1.
+            - Often known as CSI.
+            - "True positives" is the same as "hits"
+            - "False positives" is the same as "false alarms"
+            - "True negatives" is the same as "correct negatives"
+
+        References:
+            https://www.cawcr.gov.au/projects/verification/#CSI
         """
         return self.threat_score()
 
-    def peirce_skill_score(self):
+    def peirce_skill_score(self) -> xr.DataArray:
         """
-        Hanssen and Kuipers discriminant (true skill statistic, Peirce's skill score)
+        Identical to Hanssen and Kuipers discriminant and the true skill statistic
+
         How well did the forecast separate the "yes" events from the "no" events?
-        Range: -1 to 1, 0 indicates no skill. Perfect score: 1.
 
-        https://www.cawcr.gov.au/projects/verification/#HK
+        Returns:
+            An xarray object containing the Peirce Skill Score
+
+        .. math::
+            \\text{Peirce skill score} = \\frac{\\text{hits}}{\\text{hits} + \\text{misses}} - \\frac{\\text{false alarms}}{\\text{false alarms} + \\text{correct negatives}}
```

Comment:
Update to be consistent with everything else ( true positives etc)

Reply:
I get to this in the next PR, thanks for noting it but I will address it later.